downloads/swan-lake-release-notes/swan-lake-2201.8.3/RELEASE_NOTE.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: ballerina-left-nav-release-notes
+title: Swan Lake Update 8 (2201.8.3) 
+permalink: /downloads/swan-lake-release-notes/2201.8.3/
+active: 2201.8.3
+redirect_from: 
+    - /downloads/swan-lake-release-notes/2201.8.3
+    - /downloads/swan-lake-release-notes/2201.8.3/
+    - /downloads/swan-lake-release-notes/2201.8.3-swan-lake/
+    - /downloads/swan-lake-release-notes/2201.8.3-swan-lake
+---
+
+## Overview of Ballerina Swan Lake Update 8 (2201.8.3)
+
+<em>Swan Lake Update 8 (2201.8.3) is the third patch release of Ballerina 2201.8.0 (Swan Lake Update 8) and it includes a new set of bug fixes to the runtime and developer tooling.</em>
+
+## Update Ballerina
+
+Run the command below to update your current Ballerina installation directly to 2201.8.3 by using the [Ballerina Update Tool](/learn/update-tool/) as follows.
+
+```
+$ bal dist pull 2201.8.3
+```
+
+## Install Ballerina
+
+If you have not installed Ballerina, download the [installers](/downloads/#swanlake) to install.
+
+## Runtime updates
+
+### Bug fixes
+
+To view bug fixes, see the [GitHub milestone for 2201.8.3 (Swan Lake)](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A2201.8.3+label%3AType%2FBug+is%3Aclosed+label%3ATeam%2FjBallerina).
```

Comment:
```suggestion
To view bug fixes, see the [GitHub milestone for 2201.8.3 (Swan Lake)](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A2201.8.3+label%3AType%2FBug+is%3Aclosed+label%3ATeam%2FjBallerina).
```
This milestone link is empty.

Reply:
I think there is no difference with suggested changes